lib/jara/releaser.rb

```diff
@@ -25,7 +25,14 @@ def initialize(environment, bucket_name=nil, options={})
       @file_system = options[:file_system] || FileUtils
       @s3 = options[:s3]
       @logger = options[:logger] || IoLogger.new($stderr)
-      @branch = @environment == 'production' ? 'master' : @environment
+      @branch = if options[:branch]
+        raise JaraError, 'Production must be released from master' if @environment == 'production'
+        options[:branch]
+      elsif @environment == 'production'
+        'master'
+      else
+        @environment
+      end
```

Comment:
I would have written this as

```ruby
if @environment == 'production'
  if options[:branch]
    raise JaraError, 'Production must be released from master'
  end
  @branch = 'master'
else
  @branch = options[:branch] || @environment
end
```

It's easier to read since it does one thing at a time, but using the same number of lines.